src/i18n/locales/pt-br.json

```diff
@@ -0,0 +1,99 @@
+{
+    "blog.title": "Blog Node.js",
+    "blog.description": "As últimas noticias, tutoriais, recursos e exemplos do NodeJS.",
+    "blog.authors.list.title.by": "por",
+    "blog.authors.list.title.and": "e",
+    "components.downloadAdditional.title": "Downloads Adicionais",
+    "components.downloadHeader.navigation.activeSection": "downloads",
+    "components.downloadHeader.navigation.title": "Downloads",
+    "components.downloadHeader.navigation.nodeVersion": "{lts, select, true {ÚLTIMA VERSÃO LTS}other {VERSÃO ATUAL}} {nodeVersion}",
```

Comment:
What's the difference between these two statements? (Ultima != Atual)?

Reply:
It'd be the exact difference between "latest" (Última) and "current" (Atual) and to keep this statement coercive with the English one, I opted to keep it that way.

Reply:
I'm definitely not native Brazilian, just found that curious. Thanks for clarifying 😃

Reply:
Thank you for asking 😉